### PR TITLE
Add trackScrollEvents setting to docs

### PR DESCRIPTION
**Pull request by imurchie**
## Proposed changes

Add docs for `trackScrollEvents` setting.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by jlipps — COMMENTED** [review]
good, but should we mention the features that rely on this cap to make it clear?

**Comment by imurchie** [comment]
That would be great! I have no idea how this is used or for what. I looked around for some sort of documentation but found none.

**Comment by jlipps** [comment]
It is used to populate the `lastScrollData` field on the `getSession` response. This field enables you to know how far an android view scrolled in reality (since reliable pixel by pixel scrolling is not possible)

**Comment by imurchie** [comment]
Alrighty. A little more detail. Let me know if there is more to it, @jlipps !

File: docs/en/advanced-concepts/settings.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ Note that the actual commands you would use in your test script differ based on
...
 |`normalizeTagNames`| Translate all class names used as XML tags to the limited set of ASCII characters supported by Apache Harmony library. Used by default in Android to avoid possible XML parsing exceptions caused by XPath lookup. The translation is based on [junidecode](https://github.com/gcardone/junidecode). This prevents [this error case](https://github.com/appium/appium/issues/11854). Defaults to `false`. | `false` or `true` |
 |`shutdownOnPowerDisconnect`| Shutdown the server through the broadcast receiver on [ACTION_POWER_DISCONNECTED](https://developer.android.com/reference/android/content/Intent.html#ACTION_POWER_DISCONNECTED). Defaults to `true` | `false` or `true` |
+|`trackScrollEvents`| Turn on or off the tracking of scroll events as they happen. If `true`, a field, `lastScrollData`, is added to the results og `getSession`, which can then be used to check on scroll progress. Turning this feature off significantly increases touch action performance. Defaults to `true` | `true` or `false` |
```
Comment (KazuCocoa) [inline docs/en/advanced-concepts/settings.md]: s/the results **og** `getSession`/the results **of** `getSession`/ ?

Comment (imurchie) [reply docs/en/advanced-concepts/settings.md]: **of**, natürlich.

**Review by KazuCocoa — APPROVED** [review]
looks 👍 to me

**Pull request merged**